x-pack/plugins/task_manager/server/integration_tests/managed_configuration.test.ts

```diff
@@ -6,19 +6,25 @@
  */
 
 import sinon from 'sinon';
-import { savedObjectsRepositoryMock } from '../../../../../src/core/server/mocks';
+import {
+  elasticsearchServiceMock,
+  savedObjectsRepositoryMock,
+} from '../../../../../src/core/server/mocks';
 import { SavedObjectsErrorHelpers, Logger } from '../../../../../src/core/server';
 import { ADJUST_THROUGHPUT_INTERVAL } from '../lib/create_managed_configuration';
 import { TaskManagerPlugin, TaskManagerStartContract } from '../plugin';
 import { coreMock } from '../../../../../src/core/server/mocks';
 import { TaskManagerConfig } from '../config';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { elasticsearchClientMock } from 'src/core/server/elasticsearch/client/mocks';
```

Comment:
The core team doesn't recommend accessing underlying paths. It is best to use the `elasticsearchServiceMock` that is imported above and use `elasticsearchServiceMock.createErrorTransportRequestPromise`.

```suggestion
```